ibm/service/iamidentity/resource_ibm_iam_trusted_profile_template.go

```diff
@@ -122,7 +122,7 @@ func ResourceIBMTrustedProfileTemplate() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"iam_id": {
 										Type:        schema.TypeString,
-										Required:    true,
+										Computed:    true,
```

Comment:
Can we mark it as optional and then change back to computed after 2 release in b/w 
Directly removing as Required breaks the customer